app/src/main/java/com/android/voyageur/ui/overview/Overview.kt

```diff
@@ -209,8 +220,46 @@ fun TripItem(
                             ))
                     DisplayParticipants(trip, userViewModel)
                   }
+              Box(modifier = Modifier.align(Alignment.Top)) {
+                IconButton(
+                    onClick = { isExpanded = !isExpanded },
+                    modifier = Modifier.testTag("expandIcon_${trip.name}")) {
+                      Icon(
+                          imageVector = Icons.Default.MoreVert,
+                          contentDescription = if (isExpanded) "Collapse" else "Expand")
+                    }
+                DropdownMenu(
+                    expanded = isExpanded,
+                    onDismissRequest = { isExpanded = false },
+                    modifier = Modifier.background(MaterialTheme.colorScheme.secondaryContainer)) {
+                      DropdownMenuItem(
+                          onClick = {
+                            isExpanded = false
+                            showDialog = true
+                          },
+                          text = { Text("Delete") },
+                          modifier = Modifier.testTag("deleteMenuItem_${trip.name}"))
+                    }
+              }
             }
       })
+  // Confirmation Dialog
+  if (showDialog) {
+    AlertDialog(
+        onDismissRequest = { showDialog = false },
+        title = { Text(text = "Remove Trip") },
+        text = { Text("Are you sure you want to remove \"${trip.name}\" from your trips?") },
+        confirmButton = {
+          TextButton(
+              onClick = {
+                tripsViewModel.deleteTripById(trip.id)
```

Comment:
Users do not receive any visual feedback upon successfully deleting a trip, we should use the OnSuccess parameter of the deleteTripById to display a message saying that we indeed deleted that trip.

Reply:
Also, what happens if the deleteTripbyId fails ? Here the deletion action assumes success without verifying the result from deleteTripById. We should look into error handling.

Reply:
You're right, it's better to take into account all cases. Also, for better UX, I added a toast to be displayed when the trip is successfully deleted.